packages/renderer/src/stores/explore-features.ts

```diff
@@ -0,0 +1,54 @@
+/**********************************************************************
+ * Copyright (C) 2023 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+import type { Writable } from 'svelte/store';
+import { writable } from 'svelte/store';
+
+import type { ExploreFeature } from '/@api/explore-feature';
+
+import { EventStore } from './event-store';
+
+const windowEvents = ['explore-features-loaded', 'provider-change', 'provider-container-connection-update-status'];
+const windowListeners = ['update-explore-features'];
+
+let readyToUpdate = false;
+
+export async function checkForUpdate(eventName: string): Promise<boolean> {
+  if ('explore-features-loaded' === eventName) {
+    readyToUpdate = true;
```

Comment:
Looks to me that once the event is received we will periodically poll the backend for the values